src/Sql/Sql/Database Backup/Cmdlet/SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy.cs

```diff
@@ -34,6 +34,14 @@ public class SetAzureRmSqlDatabaseBackupShortTermRetentionPolicy : AzureSqlDatab
         [ValidateNotNullOrEmpty]
         public int RetentionDays{ get; set; }
 
+        /// <summary>
+        /// Gets or sets differential backup interval hours.
+        /// </summary>
+        [Parameter(Mandatory = true,
+            Position = 4,
+            HelpMessage = "The differential backup interval, in hours.")]
+        public int? DiffBackupIntervalInHours { get; set; }
```

Comment:
Please use an *optional* `int` parameter instead of a *mandatory* `int?` parameter.

Reply:
But we accept DiffBackupIntervalInHours == null.

Reply:
"Nullable" is a concept in .NET. In PowerShell, it is represented by the existance / absence of a parameter.

If SDK accepts a `int?`, you should write this in your code:
```csharp
this.managementClient.someOperation(new SomeModel() {
    DiffBackupIntervalInHours = this.IsParameterBound(c => c.DiffBackupIntervalInHours) ? DiffBackupIntervalInHours : null as int?
})

Reply:
where do I put this to?